examples/cdk-examples-typescript/hello-cdk-ecs/cdk.json

```diff
@@ -0,0 +1,42 @@
+{
+  "app": "node index",
+  "context": {
+    "availability-zones:585695036304:us-east-1": [
```

Comment:
Is it normal practice in the CDK to check in account IDs?  Normally we avoid publicly sharing account IDs...

Reply:
Nope. We should definitely delete

Reply:
Yeah this should not have been checked in.

Reply:
Do we expect customers to check this in to their repos, or should cdk.json generally be git'ignored?  Should cdk init add that to a .gitignore file?  Sounds like we need a separate issue to track that

Reply:
fyi this stuff is online for all to see